cumulusci/robotframework/tests/salesforce/TestListener.py

```diff
@@ -1,32 +1,84 @@
-"""
-    This hybrid library/listener can be used to verify
-    messages that have been logged.
+"""This hybrid library/listener can be used to verify messages that
+    have been logged and keywords have been called.
+
+    This works by listening for log messages and keywords via the
+    listener interface, and saving them in a cache. Keywords are
```

Comment:
It seems like more of a log than a cache to me (cache sounds like things would get accessed randomly, and invalidated individually), but I don't feel strongly about what it's called.

Reply:
You're right, I was being lazy. I've refactored a good bit of the code to introduce some consistency.